library-best-practices/JLBP-6.md

```diff
@@ -54,9 +54,13 @@ Recommendations:
   - Corollary 2: Don't fork an artifact owned by someone else and publish
     classes with the same fully-qualified classnames.
 
-Extended discussion
+Extended discussion with example scenario
 -------------------
 
+The recommendations are derived from our experience around renaming package and/or Maven ID.
```

Comment:
I'm not sure this is needed

Reply:
These additions line 57 - 62 are to clarify whether this section is
- to support the recommendations with examples (<- I believe this is it)
- to introduce exceptions of the recommendation
- to explain other recommendations
etc.

I wanted this clarification, because this section is long. Having a simple paragraph would help readers decide whether they should read it or can skip it.